src/commands/aem/rde/setup.js

```diff
@@ -397,11 +419,21 @@ class SetupCommand extends BaseCommand {
   }
 }
 
+SetupCommand.enableJsonFlag = false;
+
 Object.assign(SetupCommand, {
   description: 'Setup the CLI configuration necessary to use the RDE commands.',
```

Comment:
It would be more in keeping with the style currently employed to add the `enableJsonFlag` via the `Object.assign()` call (in all places, including the `BaseCommand`). Of course, ideally we would move to typescript sooner than rather than later, so we can follow the oclif docs 1:1.

```suggestion
Object.assign(SetupCommand, {
  description: 'Setup the CLI configuration necessary to use the RDE commands.',
  enableJsonFlag: false,
```